src/shogun/machine/LinearMachine.h

```diff
@@ -66,12 +66,20 @@ class CLinearMachine : public CMachine
 		/** default constructor */
 		CLinearMachine();
 
+		/** Constructor
+		 *
+		 * @param src_bias_enable new bias_enable
+		 */
+		CLinearMachine(bool src_bias_enable);
```

Comment:
why the src_ prefix?
just call this thing `compute_bias` rather than `m_compute_bias`